scripts/ci-build-subprojects.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+## Builds all RChain subprojects 
+
+# Set BASH environment so it will properly fail throwing exit code
+set -euxo pipefail
+
+## Set java options
+JAVA_OPTS=-Dfile.encoding=UTF8 
```

Comment:
Not needed with #328, is it?

Reply:
No. I'll remove it. I left it in just cuz. Have a bad habit of leaving more information and code in than less. I need to work o that.